src/prebuilt/wasm2c.include.h

```diff
@@ -17,6 +17,8 @@ const char SECTION_NAME(top)[] =
 "#define WASM_RT_PASTE(x, y) WASM_RT_PASTE_(x, y)\n"
 "#define WASM_RT_ADD_PREFIX(x) WASM_RT_PASTE(WASM_RT_MODULE_PREFIX, x)\n"
 "\n"
+"#define WASM_CURR_ADD_PREFIX(x) WASM_RT_PASTE(WASM_CURR_MODULE_PREFIX, x)\n"
```

Comment:
This separates a prefix that applies to the wasm runtime (shared across all modules) and a prefix that is unique to this module (since applications can have more than one wasm module). Will add to docs